### ISSUE TITLE
page request: lastcomm

### ISSUE THREAD

**Issue by StewAlexanderACC** [tldr-pages/tldr#3086]
[Lastcomm Man Page](http://man7.org/linux/man-pages/man1/lastcomm.1.html)

* Shows the last commands run by any user:
```
support@ntopng:/$ lastcomm |less
bash                   Debian-s __         0.00 secs Mon May 20 09:57
bash              F    Debian-s __         0.00 secs Mon May 20 09:57
grep                   Debian-s __         0.00 secs Mon May 20 09:57
apt-get                Debian-s __         1.09 secs Mon May 20 09:57
http                 X Debian-s __         0.00 secs Mon May 20 09:57
dpkg                   Debian-s __         0.00 secs Mon May 20 09:57
dpkg                   Debian-s __         0.00 secs Mon May 20 09:57
clear                  support  pts/0      0.00 secs Mon May 20 09:57
```
* Using "***lastcomm {username}***" limits the commands to a specific user:
```
support@ntopng:/$ lastcomm support
lastcomm             X support  pts/0      0.00 secs Mon May 20 09:57
less                   support  pts/0      0.00 secs Mon May 20 09:57
clear                  support  pts/0      0.00 secs Mon May 20 09:57
lastcomm               support  pts/0      0.08 secs Mon May 20 09:57
clear                  support  pts/0      0.00 secs Mon May 20 09:57
sa                     support  pts/0      0.00 secs Mon May 20 09:54
clear                  support  pts/0      0.00 secs Mon May 20 09:54
grep                   support  pts/0      0.00 secs Mon May 20 09:47
sa                     support  pts/0      0.01 secs Mon May 20 09:47
less                   support  pts/0      0.08 secs Mon May 20 09:43
support@ntopng:/$
```
* Using "***lastcomm {command name}***" gives the last time a command was issued and by whom:
```
support@ntopng:/$ lastcomm ac
ac                     support  pts/0      0.00 secs Mon May 20 09:21
ac                     support  pts/0      0.00 secs Mon May 20 09:19
ac                     support  pts/0      0.00 secs Mon May 20 09:18
support@ntopng:/$
```

**Comment by sbrl** [comment]
Thanks for requesting this page, @StewAlexanderACC! Do you want to have a go yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)

**Comment by thenarcissist** [comment]
I want to take it up if it's still up. (Will do this weekend.)

**Comment by StewAlexanderACC** [comment]
@thenarcissist I've been particularly busy, you are welcome to it, and thank you

**Comment by sbrl** [comment]
Go ahead @thenarcissist :D

**Comment by schneiderl** [comment]
Closing this as it was resolved on #3333.

Thank you @lmazardo for the page.
